### PR TITLE
Add localstack container when it test-others

### PR DESCRIPTION
**Pull request by whoisxx**
## Proposed changes

- Add localstack container to test-others for local S3, DynamoDB
- Update build_base image to communicate localstack container

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -63,7 +68,6 @@ commands:
               echo "this is not RC version"
             fi
             echo "export KLAYTN_VERSION=$(go run build/rpm/main.go version)" >> $BASH_ENV
-
```
Comment (ehnuje) [inline .circleci/config.yml]: Please remove unnecessary line removal.

File: .circleci/config.yml [hunk 2]
```diff
@@ -268,10 +274,18 @@ jobs:
...
-    executor: test-executor
+    executor: test-others-executor
     steps:
       - install-ssh-alpine
       - checkout
+      - run:
+          name: "Wait until localstack container running"
+          command: |
+            timeout -t 60 sh -c \
+            'until curl --silent --fail "http://localhost:4572"; do
```
Comment (ehnuje) [inline .circleci/config.yml]: I can't find `localstack start` command. Do we need to start localstack manually or we can just use it after this PR is merged?

File: .circleci/config.yml (hunk 1 above)
Comment (whoisxx) [reply .circleci/config.yml:66]: which line?

File: .circleci/config.yml (hunk 2 above)
Comment (whoisxx) [reply .circleci/config.yml:285]: `test-others` use klaytn/build_base with localstack container. so you can see it(klaytn/build_base image) call localhost:4572 is open which is used for localstack container for S3

File: .circleci/config.yml (hunk 1 above)
Comment (ehnuje) [reply .circleci/config.yml:66]: This empty line(=line66) has been removed

Comment (whoisxx) [reply .circleci/config.yml:66]: Right. It removed now :)

Comment (ehnuje) [reply .circleci/config.yml:66]: Oh, it is intended. I understood.

**Pull request merged**